spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/RNN.scala

```diff
@@ -165,14 +171,6 @@ class RNN(
       PropKind.ForwardTraining
     }
 
-    /**
```

Comment:
I think those comments should not be deleted.

Reply:
Changed back